modules/imap/output_modules.php

```diff
@@ -202,6 +202,13 @@ protected function output() {
                             } catch (Exception $e) {}
                             $txt .= '<tr class="header_'.$fld.'"><th>'.$this->trans($name).'</th><td>'.$this->html_safe($value).'</td></tr>';
                         }
+                        elseif ($fld == 'reply-to') {
+                            if (str_replace(['<','>'], '', $headers['From']) !== $value) {
```

Comment:
@Yannick243 according to https://datatracker.ietf.org/doc/html/rfc4021#section-2.1.4 the Reply-To header field can hold multiple emails. Testing it this way will no work then.

Also there is a function in modules/imap/functions.php, process_address_fld which helps to us get label and email from a given header field as the Reply-To field can have format `Some Name <abc@cdf.com>, axy@domain.fr`